src/components/FullCourt/index.tsx

```diff
@@ -35,7 +35,6 @@ const FullCourt = () => {
   const canvasStates = canvasControl.canvasStates;
 
   useEffect(() => {
-    console.log(canvasStates.selectedColor);

```

Comment:
That's a great approach, but I feel it's unrelated to the content of this card. Perhaps we could create a separate card later to address these irrelevant pieces of code.